Eip 5792 extension

technophile-04 opened the pull request:
### Description

Fixes #6

rin-st commented on extension/packages/nextjs/app/eip-5792/_components/EIp5792Example.tsx:
```diff
@@ -0,0 +1,157 @@
+          isPaymasterSupported && Boolean(paymasterURL)
+            ? {
+                paymasterService: {
+                  url: `${paymasterURL}/api/paymaster`,
```
why to add `/api/paymaster` here?
I tried to use paymaster URL from coinbase docs https://portal.cdp.coinbase.com/products/bundler-and-paymaster. It works by default, but doesn't work with `/api/paymaster`. Default URL is `https://api.developer.coinbase.com/rpc/v1/base-sepolia/<YOUR ID>`

rin-st reviewed (commented):
Great extension! 👍 

Tested it and everything works nicely! Just added one question regarding paymaster

rin-st commented:
Plus fixed typo [96a1005](https://github.com/scaffold-eth/create-eth-extensions/pull/25/commits/96a100515964465eceb77b5acbc4bd524231482b)

Pabl0cks reviewed (commented):
It's looking great but I'm having a hard time connecting the coinbase smart wallet (only manage to connect throught coinbase extension but seems it's not "smart" haha).

Tried on different devices but will try more tomorrow until I make it work properly 🙌

Deleted installation generic text in https://github.com/scaffold-eth/create-eth-extensions/pull/25/commits/1f063ecf39cdfe86b1a1b580ab977f13eff1d00a

Pabl0cks commented:
For some reason coinbase smart wallet was not working when connecting from laptop to PC, it was not letting me choose smart wallet, only extension. When running directly on localhost from laptop it did let me choose smart wallet 👌

First time that connected to smart wallet I received a "This request method is not supported" but when disconnected and connected again it worked nicely the rest of times 🙏

Pabl0cks commented on extension/packages/nextjs/app/eip-5792/page.tsx:
```diff
@@ -0,0 +1,32 @@
+          <span className="block text-4xl font-bold">EIP-5792 example</span>
+        </h1>
+        <p className="my-0 text-center">
+          This extension demonstrates on how to use{" "}
```
"demonstrates how to use" may feel more natural?

Pabl0cks commented on extension/packages/nextjs/app/eip-5792/page.tsx:
```diff
@@ -0,0 +1,32 @@
+            EIP-5792
+          </a>{" "}
+          wallet capabilities. This EIP introduces new JSON-RPC methods for communication between apps and wallets.
+          Which allows for more advanced interactions like submitting multiple onchain calls as part of a single
```
What about "This" instead of "Which"?

Pabl0cks commented on extension/packages/nextjs/app/eip-5792/page.tsx:
```diff
@@ -0,0 +1,32 @@
+          transaction or sponsoring users transactions via ERC-4337 paymasters.
+        </p>
+        <p className="my-0 text-center">
+          Below you can see an example of interacting with a contract using EIP-5792 compliant wallet (coinbase smart
+          wallet). We can change the state of the contract by setting new greetings or incrementing the counter
```
Ultra nitpick: (Coinbase Smart Wallet)

Pabl0cks reviewed (approved):
Awesome job @technophile-04 !! Working nicely with both hardhat and foundry installations.

Needed to use Rinat's hint, if not I was getting "This request method is not supported" message when trying to Batch transaction.

Added a couple of nitpick comments on the introduction text, ignore if you feel they don't make sense,  I'm not native! 🙌

technophile-04 commented:
Tysm guys!! Just pushed some commits addressing all the comments 🙌 also updated foundry deployments inline with latest foundry updates. 

So `npx` will break now for this extension until we publish new version of CLI (we will soon publish the new version) and we need to update he README of this with EIP-5792 extension and add this to curated extension in CLI

technophile-04 commented:
Merging this thanks all! We will be soon releasing the version
